src/lib.rs

```diff
@@ -248,16 +266,10 @@ impl Plugin for OutlinePlugin {
             .get_sub_graph_mut(bevy::core_pipeline::core_3d::graph::NAME)
             .unwrap();
         draw_3d_graph.add_node(OUTLINE_PASS_NODE_NAME, node);
-        draw_3d_graph.add_slot_edge(
-            draw_3d_graph.input_node().id,
-            bevy::core_pipeline::core_3d::graph::input::VIEW_ENTITY,
-            OUTLINE_PASS_NODE_NAME,
-            OutlineNode::IN_VIEW,
-        );
 
         // Run after main 3D pass, but before UI psss
         draw_3d_graph.add_node_edge(
-            bevy::core_pipeline::core_3d::graph::node::MAIN_PASS,
+            bevy::core_pipeline::core_3d::graph::node::END_MAIN_PASS,
```

Comment:
I am pretty sure using `END_MAIN_PASS` is identical to the old `MAIN_PASS` in terms of render graph, so this might be a different problem. I'm not sure what that is though.